sdk/servicebus/service-bus/test/stress/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # internal users should provide MCR registry to build via 'docker build . --build-arg REGISTRY="mcr.microsoft.com/mirror/docker/library/"'
 # public OSS users should simply leave this argument blank or ignore its presence entirely
 ARG REGISTRY=""
-FROM ${REGISTRY}node:12-alpine
+FROM ${REGISTRY}node:lts
```

Comment:
I don't see "lts" in the tags here.
https://mcr.microsoft.com/en-us/product/mirror/docker/library/node/tags

Reply:
Odd, are we not syncing everything from https://hub.docker.com/_/node/tags ?

Reply:
Haven't tested.
If it isn't working, there is an option to request the tags, by emailing them.

Reply:
Being a little explicit would probably be good anyways, since it makes it a bit simpler to know what major-version runtime you tested against.

Reply:
Maybe that's the reason they weren't syncing "lts" :P

Reply:
updated to 18